packages/devtools/lib/src/performance/performance.dart

```diff
@@ -325,24 +325,24 @@ class PerfColumnSelf extends Column<PerfData> {
   bool get numeric => true;
 
   @override
-  dynamic getValue(PerfData item) => item.self;
+  dynamic getValue(PerfData dataObject) => dataObject.self;
 
   @override
   String render(dynamic value) => percent2(value);
 }
 
 class PerfColumnMethodName extends Column<PerfData> {
-  PerfColumnMethodName() : super('Method', wide: true);
+  PerfColumnMethodName() : super('Method', percentWidth: 0.8);
```

Comment:
.8 is used in a few places. Maybe make it defaultWideColumnPercent?

Reply:
done. Also changed to 1.0 everywhere so the wide column will take up all of the available remaining width available.